src/libpmemfile-posix/inode.c

```diff
@@ -128,6 +129,10 @@ inode_ref(PMEMfilepool *pfp, TOID(struct pmemfile_inode) inode,
 		os_rwlock_init(&vinode->rwlock);
 		vinode->tinode = inode;
 		vinode->inode = PF_RW(pfp, inode);
+
+		void *key = (void *) vinode->inode;
+		vinode->stat_block_cache = cache_get(key);
```

Comment:
Initially this value will be invalid, even if file is not empty. Think what will happen in truncate or write.